styles.css

```diff
@@ -0,0 +1,213 @@
+.container {
+  width: 100vw;
```

Comment:
I like that you used vw/vh so the main container takes up the full viewport height/width